challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -1043,9 +1045,9 @@
             "}",
             "/**",
```

Comment:
This message could show something closer to the actual array that is logged. Could be a good way to reinforce the fact that template literals are really just another way to create strings:

```
            "/**",
            " * makeList(result.failure) should return:",
            " * [ \"<li class='text-warning'>no-var</li>\",",
            " *   \"<li class='text-warning'>var-on-top</li>\",",
            " *   \"<li class='text-warning'>linebreak</li>\" ]",
            " **/",
```

This one could definitely go either way, though. Let's go with what you think is better in this case.

Reply:
it seems a bit misleading to show an array of strings rather than an array of template literals as an example of correct return on this challenge